modules/ding_permissions/ding_permissions.module

```diff
@@ -1011,3 +1013,14 @@ function ding_permissions_secure_permissions($role) {
     return $permissions[$role];
   }
 }
+
+/**
+ * Implements hook_page_alter().
```

Comment:
There was a problem with toolbars (core toolbar and admin_menu toolbar) when the user with role "administrator" and uid=1 was logged in, both toolbars was displayed. Through standard permissions system it was not possible to disable core toolbar for administrator role, we were forced to unset this on page build.